db/schema.rb

```diff
@@ -30,6 +30,14 @@
     t.index ["priority", "run_at"], name: "delayed_jobs_priority"
   end
 
+  create_table "tweets", force: :cascade do |t|
+    t.string "body"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.bigint "user_id"
```

Comment:
Do we want tweets to be able to be created if they do not have a user? Should this have a null: false as well?

Reply:
yes, i also added null: false on the body, which (if i remember correctly) just means that a tweet can't be blank...right?